pep-0501.txt

```diff
@@ -383,26 +504,98 @@ errors all raise SyntaxError.
 
 Unmatched braces::
 
-  >>> i'x={x'
+  >>> t'x={x'
     File "<stdin>", line 1
-  SyntaxError: missing '}' in interpolation expression
+        t'x={x'
+           ^
```

Comment:
```suggestion
```

Something evidently went wrong with the suggestion application, and this got added twice